src/components/v2/pinInput.tsx

```
@@ -55,13 +57,13 @@ export const PinInput = ({
   };
 
   return (
-    <CyDView className='flex-row justify-center'>
+    <CyDView className={className}>
       {Array.from({ length }, (_, index) => (
         <CyDTextInput
           key={index}
           ref={el => (inputRefs.current[index] = el)}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)